### PR TITLE
[IMPROVE] User gets UI feedback when message is pinned or unpinned

### LINKED ISSUE #13869: No user feedback when meassage is pinned or unpinned
### Description:

There is no user feedback telling the user that his message has been pinned or unpinned

### Steps to reproduce:

1. Select a message to be pinned or unpinned if already pinned
2. Click on three dots in the message (Open the message actions menu)
3. Click on pin message or unpin message.

### Expected behavior:

The user should get UI feedback that his message has been pinned or unpinned otherwise failure is shown.

### Actual behavior:

No feedback is given to the user

### PR DESCRIPTION
**Pull request by ashwaniYDV**
Closes #13869
  ### This PR is continuation of the closed PR https://github.com/RocketChat/Rocket.Chat/pull/13870 where some changes were requested.
**When message is pinned.**
Screenshot from 2019-03-23 14-19-54

**When message is unpinned.**
Screenshot from 2019-03-23 14-20-05

### CODE REVIEW

**Comment by ashwaniYDV** [comment]
@ggazzo Please review.

**Comment by ashwaniYDV** [comment]
@ggazzo Can you please review this PR.

**Comment by ashwaniYDV** [comment]
@ggazzo Would you please review.

File: app/message-pin/client/pinMessage.js [hunk 1]
```diff
@@ -1,19 +1,25 @@
...
 import { ChatMessage, Subscriptions } from '../../models';
 
 Meteor.methods({
 	pinMessage(message) {
 		if (!Meteor.userId()) {
+			toastr.error(TAPi18n.__('error-pinning-message'));
 			return false;
 		}
 		if (!settings.get('Message_AllowPinning')) {
+			toastr.error(TAPi18n.__('error-pinning-message'));
```
Comment (MartinSchoeler) [inline app/message-pin/client/pinMessage.js]: We shouldn't throw the same error for different motives, here we should say something like `pinning is not allowed`

File: app/message-pin/client/pinMessage.js [hunk 2]
```diff
@@ -24,14 +30,18 @@ Meteor.methods({
 	},
 	unpinMessage(message) {
 		if (!Meteor.userId()) {
+			toastr.error(TAPi18n.__('error-unpinning-message'));
 			return false;
 		}
 		if (!settings.get('Message_AllowPinning')) {
+			toastr.error(TAPi18n.__('error-unpinning-message'));
```
Comment (MartinSchoeler) [inline app/message-pin/client/pinMessage.js]: Same here

**Comment by ashwaniYDV** [comment]
@MartinSchoeler I have done the requested changes.

**Pull request merged**